tests/models/profile.test.js

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
+import { sequelize, dataTypes, checkModelName, checkPropertyExists}  from 'sequelize-test-helpers';
+import ProfileModel from '../../server/models/profile';
+import UserModel from '../../server/models/user';
+
+describe('Profile properties', () => {
+  const Profile = ProfileModel(sequelize, dataTypes);
+  const profile = new Profile();
+  checkModelName(Profile)('Profile');
```

Comment:
You can use the it() block for better readability

Reply:
it is already readable as it is, after the test is run a comment that the model exits is displayed on the console.